src/setting.h

```diff
@@ -0,0 +1,28 @@
+#ifndef SETTING_H
+#define SETTING_H
+
+#include <QObject>
+
+class Setting : public QObject
+{
+    Q_OBJECT
+public:
+    explicit Setting(QObject *parent = nullptr);
+    Q_PROPERTY(bool getIsElysia READ getIsElysia NOTIFY isElysiaChanged FINAL)
```

Comment:
This closer matches how we did the QProperties for packet fields, I think isElysia is a better name to use for the qml
```suggestion
    Q_PROPERTY(bool isElysia READ isElysia WRITE setIsElysia NOTIFY isElysiaChanged FINAL)
```